storage/innobase/buf/buf0dblwr.cc

```diff
@@ -514,6 +599,104 @@ buf_dblwr_init_or_load_pages(
 		page += univ_page_size.physical();
 	}
 
+	err = buf_parallel_dblwr_make_path();
+	if (err != DB_SUCCESS)
+		return(err);
+
+	ut_ad(parallel_dblwr_buf.file == OS_FILE_CLOSED);
+	bool success;
+	parallel_dblwr_buf.file
```

Comment:
The file is read once here, sequentially, and closed. Probably os_file_setnocache makes sense too